Makefile

```diff
@@ -209,6 +209,33 @@ check-kuttl:
 	${PGO_KUBE_CLIENT} ${KUTTL_TEST} \
 		--config testing/kuttl/kuttl-test.yaml
 
+# Set reasonable default for KUTTL_ env vars for templating
+KUTTL_PG_VERSION ?= 14
+KUTTL_PG_UPGRADE_FROM_VERSION ?= 13
+KUTTL_PSQL_CLIENT_IMAGE ?= registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-14.1-0
+
+# TODO: When I attempted to loop through the directories and find the yaml files,
+# due to how makefile expands variables, each file would end up in each directory.
+# So this implementation loops through to create the folders under generated
+# and then loops through all the yaml files, using a `sed` substition to place them
+.PHONY: generate-kuttl
+generate-kuttl:
+	for SRCDIR in $(shell ls testing/kuttl/source); do \
+		mkdir -p testing/kuttl/generated/$${SRCDIR}; \
```

Comment:
Makes sense, I'll add it to clean, but I also think it might make sense to add a line here cleaning up the folder because it seems like something we wouldn't want to let users (us) forget to do.